### ISSUE TITLE
Move/Remove the Great Prophet panel in GP view to the last slot

### ISSUE THREAD

**Issue by chaorace** [chaorace/cqui#40]
Barring the very early game, great prophets don't really bear any relevance to someone visiting the GP view. Because of this, it makes sense to move the great prophet panel to the last slot on the GP view... and to possibly even hide it entirely once all available great prophets have been created

**Comment by jacks0nX** [comment]
Moved:

![image](https://cloud.githubusercontent.com/assets/2789832/20031948/b59b7014-a380-11e6-919a-aa845bbff26b.png)

Removed when religion already founded:

![image](https://cloud.githubusercontent.com/assets/2789832/20031962/fbd75dcc-a380-11e6-8b65-d7d81f221847.png)

**Comment by chaorace** [comment]
Yep, that looks pretty good!

**Comment by zyxpsilon** [comment]
I'll be adding a new mockup structure (with precise calibration details) for each Panels.. soon. But we'll still have to wait for the SDK assets to adapt **their** modal principles to **our** needs, i gather. ;)

**Comment by zyxpsilon** [comment]
Soooo.. here's that new structure -- although i had explained it all in another "post" that was deleted, it should be obvious how this could spare plenty of pixels on the whole framework.

![panel_before-after](https://cloud.githubusercontent.com/assets/23318465/20074787/b7a44d7e-a4ff-11e6-950f-589375cbc4c3.png)

**Comment by jacks0nX** [comment]
Looks great, this definitely is more pleasing to see than the current solution, in my opinion. Would maybe still include a "Progress" above the progress bars though, looks kind of empty down there.

**Comment by zyxpsilon** [comment]
Yep.. maintaining the terms "PROGRESS" and "With Patronage" above the costs duo would be preferable too i guess.
The trick i gather will be to calibrate the Modal dispatch and possibly create a new DDS image (BLP packaged) to fit the narrower framing -- and for that, we'd need the SDK for sure. Unless you'd have some magic indirect solution.
Don't forget the big round GP faces can also be discarded or re-sized if need be.
Good luck! :)

**Comment by jacks0nX** [comment]
Opened a new issue for the redesign.

**Comment by vttale** [comment]
Do the purple dots above the recruit button signify anything?  If not, should they just go away?  If so, should they be more clear about what they represent?
